projects/plutus/inc/ltc2484.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <stdint.h>
+
+#include "gpio.h"
+#include "spi.h"
+#include "status.h"
+
+// commands
+// see datasheet Table 1 (p. 14) for these command values
+#define LTC2484_ENABLE (1 << 7)
```

Comment:
I think we should move these into an "internal" header file.

Reply:
Yeah, I was trying to avoid an internal-only header file, but I can move these (and the `union`) into that.

Reply:
Yeah, I think it clutters up the user-facing header a little too much.

Reply:
This is now the internal header file, and `ltc_adc.h` is the public header.